### PR TITLE
Add more tests for PERMAS

### PR DESCRIPTION
**Pull request by nilswagner**

### CODE REVIEW

**Comment by nschloe** [comment]
So, it looks like you enable tests for more cell types, and then disable some for _all_ mesh formats via
```python
        if cell_type in ["triangle6", "tetra10"]:
            pass
        else:
            assert numpy.allclose(data, mesh.cells[cell_type])
```
Better: Just don't enable the permas tests for `triangle6`, `tetra10`.

**Comment by nilswagner** [comment]
How can I disable permas tests solely for triangle6 and tetra10?

File: test/test_permas.py [hunk 1]
```diff
@@ -6,7 +6,16 @@
 
 @pytest.mark.parametrize(
     "mesh",
-    [helpers.tri_mesh, helpers.quad_mesh, helpers.tri_quad_mesh, helpers.tet_mesh],
+    [
+        helpers.tri_mesh,
+        helpers.triangle6_mesh,
```
Comment (nschloe) [inline test/test_permas.py]: > How can I disable permas tests solely for triangle6 and tetra10?

Simply leave it commented out here.

**Comment by nschloe** [comment]
Closing in favor of #491.

**Pull request closed without merging**